Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/CiscoUmbrellaReporting.py

```diff
@@ -0,0 +1,1333 @@
+import demistomock as demisto
+
+from CommonServerPython import *
+from typing import Dict, List, Optional
+from requests.auth import HTTPBasicAuth
+from datetime import datetime
+import requests
+import urllib3
+
+# Disable insecure warnings
+urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+''' CONSTANTS '''
+
+COUNT = 0
```

Comment:
The COUNT const isn't used - where did you plan to use it? (if it is necessary please change its name to DEFAULT_COUNT, if not please remove it). 

FYI - the OFFSET const wasn't used also, so I used it in the pagination function.

Reply:
@ShacharKidor Removed COUNT